httpie/utils.py

```diff
@@ -109,6 +109,15 @@ def is_expired(expires: Optional[float]) -> bool:
         for attrs in attr_sets
     ]
 
+    # HACK/FIXME: https://github.com/psf/requests/issues/5743
+    for cookie in cookies:
+        if 'expires' in cookie:
+            continue
+
+        max_age = cookie.get('max-age')
+        if max_age and max_age.isdigit():
+            cookie['expires'] = now + float(max_age)
+
```

Comment:
This would be better as a new named function with a docstring for context & clarity.